src/main/java/Bwead/Task.java

```diff
@@ -0,0 +1,47 @@
+package Bwead;
+
+/**
+ * Represents a task that has a string method and can be marked to done.
+ */
+public class Task {
+
+    private String text;
+    private boolean isDone;
+
+    /**
+     * Constructs a Task instance using a string name.
+     *
+     * @param text name of the task.
+     */
+    public Task(String text) {
+        this.isDone = false;
+        this.text = text;
+    }
+
+    public String getText() {
+        return this.text;
+    }
+
+    public void setDone(boolean isDone) {
+        this.isDone = isDone;
+    }
+
+    public String getName() {
```

Comment:
I feel this is very vague.  🫤 
Suggestion: can change it something like getDescription() because the description doesn't contain just the name.
Also this is repetitive as you already have a getText() method doing the same thing. 🤔